Update 1 (#246)
NilPointer-Software/bdfd-wiki: src/bdscript/addEmoji.md

## Usage
```
$addEmoji[name;image URL;return emoji?]
```

## Parameters
- `name`: The name of the new emoji.

> - `image URL`: The image that the emoji showcases. This needs to be a valid image URL.

Reviewer: ```suggestion
- `image URL`: The image of the new emoji. The link needs to be a valid image URL.
```
